common/components/internal-header/template.test.js

```diff
@@ -80,10 +80,15 @@ describe('Internal header component', function() {
       const $component = $('.app-header')
       const $list = $component.find('ul.app-header__navigation')
       const $items = $list.find('li.app-header__navigation-item')
-      const $firstItem = $items.find('a.app-header__link:first-child')
-      expect($items.length).to.equal(4)
-      expect($firstItem.attr('href')).to.equal('#1')
-      expect($firstItem.text()).to.contain('Navigation item 1')
+      const $anchorNavItem = $items
```

Comment:
You can probably ignore needing to change this test actually and write a separate test where you pass in a set of items without a `href` property.